e2e/playwright/snapshot-tests.spec.ts

```diff
@@ -363,14 +363,14 @@ test('extrude on each default plane should be stable', async ({
     // clear code
     await u.removeCurrentCode()
     // add makeCode('XZ')
+    await u.openAndClearDebugPanel()
     await page.locator('.cm-content').fill(makeCode(plane))
     // wait for execution done
-    await u.openDebugPanel()
     await u.expectCmdLog('[data-message-type="execution-done"]')
     await u.clearAndCloseDebugPanel()
 
     await page.getByText('Code').click()
-    await page.waitForTimeout(80)
+    await page.waitForTimeout(150)
```

Comment:
Had the screenshots fail on me again 😞

Reply:
Dang sorry, we should just revert to the original way of loading in storage state 😞